packages/material-ui-lab/src/Pagination/Pagination.js

```diff
@@ -78,17 +78,21 @@ const Pagination = React.forwardRef(function Pagination(props, ref) {
 });
 
 // @default tags synced with default values from usePagination
+
 Pagination.propTypes = {
+  // ----------------------------- Warning --------------------------------
+  // | These PropTypes are generated from the TypeScript type definitions |
+  // |     To update them edit the d.ts file and run "yarn proptypes"     |
+  // ----------------------------------------------------------------------
   /**
    * Number of always visible pages at the beginning and end.
-   * @default 1
```

Comment:
I think you need to move these to the corresponding props in `Pagination.d.ts`

Reply:
Right. For some reason I didn't think of that. Thanks for the suggestion 😄